packages/shared/src/utils/extractRelevantMetadataFromCommunity.ts

```diff
@@ -0,0 +1,52 @@
+import { graphql, readInlineData } from 'relay-runtime';
+
+import { extractRelevantMetadataFromCommunityFragment$key } from '~/generated/extractRelevantMetadataFromCommunityFragment.graphql';
+
+import { isChainEvm } from './chains';
+import { getOpenseaExternalUrlDangerouslyForCollection } from './getOpenseaExternalUrl';
+import { getObjktExternalUrlDangerouslyForCollection } from './getTezosExternalUrl';
+import { getExternalAddressLink } from './wallet';
+
+export function extractRelevantMetadataFromCommunity(
+  communityRef: extractRelevantMetadataFromCommunityFragment$key
+) {
+  const community = readInlineData(
+    graphql`
+      fragment extractRelevantMetadataFromCommunityFragment on Community @inline {
+        chain
+        contract {
+          contractAddress {
+            address
+            ...walletGetExternalAddressLinkFragment
+          }
+        }
+      }
+    `,
+    communityRef
+  );
+
+  const { contract, chain } = community;
+
+  const contractAddress = contract?.contractAddress?.address;
+  const result = {
+    contractAddress,
+    openseaUrl: '',
+    objktUrl: '',
+    externalAddressUrl: '',
+  };
+
+  if (contractAddress) {
+    result.objktUrl = getObjktExternalUrlDangerouslyForCollection(contractAddress);
```

Comment:
we only want to provide the `objktUrl` for tezos right?

i think we also want to support Prohibition and FxHash URLs, similar to how we support those in `extractRelevantMetadataFromToken`. can be a follow-up PR though as we'll want the icons to go along with it.

<img width="280" alt="image" src="https://github.com/gallery-so/gallery/assets/12162433/ce75e4d0-d713-43b7-9e12-746f8a9f6ce5">

Reply:
1. so I added that logic to check the chain where we actually display the icon but I guess it makes more sense to have that here, so will change 
2. we don't currently support sub-communities in prohibition so there is no real prohibition or fxhash URL for the Prohibition community right?
3. web currently already shows projectUrl (globe icon), will add the openseaUrl and objktUrl stuff